flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/internal/JdbcOutputSerializerTest.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.flink.connector.jdbc.internal;
+
+import org.apache.flink.api.common.ExecutionConfig;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.common.typeutils.TypeSerializer;
+import org.apache.flink.types.Row;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
```

Comment:
since in most of the places there is `org.assertj.core.api.Assertions.assertThat;` and IIRC somewhere in Flink guide it was mentioned that it's better to use `org.assertj.core.api.Assertions.assertThat;`
 could we use it here